### DISCUSSION TITLE
Eliminate zip package in version 4?

### DISCUSSION THREAD

**Discussion by CharliePoole** [nunit/nunit-console#1567] [Ideas]
@nunit/engine-team The zip package doesn't seem to contribute much and it seems that those who use it could use other packages just as easily. I'll lay out my reasoning here and I'd appreciate your feedback.

Those who use the package typically do so because they want to create a centralized installation on their systems but either don't want to use Chocolatey or are unable to do so on their platforms.

Problems with this package...
1. It's basically a dump of everything we build - even our tests - so that anyone using it has to pick the correct directories.
2. It adds about 25% to the package test time for our builds. I suspect it adds even more to the "thinking time" required to maintain our packaging system, although this is purely anecdotal.
3. A global installation can easily be created from either of the console runner nuget packages, by use of appropriate command-line options. Such an installation would then contain exactly what is needed to execute the runner. Alternatively, of course, a user unfamiliar with the nuget command-line could simpy rename the `.nupkg` file to `.zip`.

If folks see reasons to keep the zip, then I would prefer to tailor it to contain _only_ what is needed to execute the console runners.

**Comment by stevenaw** [comment]
I am generally in favor of simplifying packaging, or in this case eliminating one if the needs can be served in other ways. 

Out of curiosity, do we know what % of downloads/installs are for the ZIP package?

**Comment by CharliePoole** [comment]
Looking only at downloads from GitHub, NuGet and Chocolatey for the 3.13.3 release, I see the following stats...

| Package                      | Downloads  | Percentage |
| ----------------------------- | -----------------:| -----------------:|
NUnit.ConsoleRunner |        268,704 | 67%
NUnit.Console              |        115,379 | 29%
nunit-console-runner  (chocolatey) |          13,494 |3%
zip                                   |           5, 463 |1%

**Reply by stevenaw** [reply]
That seems quite small. 😁 Thanks for sharing those. No objections from me if we were to drop the zip package

**Comment by CharliePoole** [comment]
I'll go ahead and drop it. See issue #1582.
